<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDialog</class>
 <widget class="QDialog" name="AdminDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>439</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>468</width>
    <height>439</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>468</width>
    <height>439</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(250, 255, 155);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>330</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>481</width>
     <height>71</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>Reseteaza scorurile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>Sterge un utilizator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>adauga un cuvant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>sterge un cuvant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>adauga un admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 5px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>un utilizator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 5px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>toti utilizatorii</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>Inapoi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>Alege categoria..</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>animale</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>autori</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pictori</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>judete</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>plante</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fructe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tari</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
   <property name="maxLength">
    <number>37</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
   <property name="maxLength">
    <number>37</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
   <property name="maxLength">
    <number>37</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
   <property name="maxLength">
    <number>37</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 12px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>Salveaza cuvantul</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>210</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Linia I:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Linia II:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>280</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Linia III:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Linia IV:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdminDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdminDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
